event-adapter/README.md

```diff
@@ -0,0 +1,181 @@
+# 1. Create a cluster ensuring that Stackdriver Logging API are enabled (both reading and writing)
+```sh
+mkdir -p $GOPATH/src/k8s.io
+cd $GOPATH/src/k8s.io
+git clone git@github.com:kubernetes/kubernetes.git
+cd kubernetes
+git checkout 9aef242a4c1e423270488206670d3d1f23eaab52
+sed -i 's/logging-write/https:\/\/www.googleapis.com\/auth\/logging.admin/g' cluster/gce/config-common.sh
+make quick-release
+./cluster/kube-up.sh
+```
+You should create “k8s.io” folder to host the project. Then, clone kubernetes project on git. Checkout to the desired version and compile it.
+Stackdriver Logging API has only write permission on the default version, but we need both reading and writing.
+So, you can modify "kubernetes/cluster/gce/config-common.sh" replacing "logging-write" with "https://www.googleapis.com/auth/logging.admin".
+Now, you can create the cluster.
+Call "kube-up.sh"script, inside “cluster” folder,  to create the cluster.
+
+
+# 2. Install the API
+
+Clone the project "https://github.com/GoogleCloudPlatform/k8s-stackdriver" into your GOPATH.
+```sh
+mkdir -p $GOPATH/src/github.com/GoogleCloudPlatform
+cd $GOPATH/src/github.com/GoogleCloudPlatform
+git clone git@github.com:GoogleCloudPlatform/k8s-stackdriver.git
+git checkout 692c11f5395b3b1d321d8384cde56a4112b5696d
+cd k8s-stackdriver/event-adapter
+```
+Now, you need to set it to work with your project. You should define a variable that identify your project GCP_PROJECT=myProject.
+```sh
+sed -i 's/google-containers/'"$GCP_PROJECT"'/g’adapter.yaml
+make push GCP_PROJECT
+kubectl create -f adapter.yaml --validate=false
+```
+The adapter.yaml file contains the destination from which you will retrieve the image, Makefile instead will contain the place where you are going to push the image.
+Everytime you change where you are going to push or to pull the image, you should change the other one accordingly.
+Calling "make push", it will compile the code and push the image.
+Then call "kubectl create -f adapter.yaml --validate=false", it will require a bit (~1 minute) to install everything.
+You can check everything is installed:
+```sh
+kubectl api-versions | grep v1events/v1alpha1
+```
+If you need to re-install it, then you will have to delete it using "kubectl delete -f adapter.yaml".
+```sh
+kubectl delete -f adapter.yaml
+make push GCP_PROJECT
+kubectl create -f adapter.yaml --validate=false
+```
+
+# 3. Create a proxy
+Create a proxy, you can choose the port on which it will serve:
```

Comment:
"Create a proxy so you can conveniently query API Server from your workstation. You can specify the port on which it will serve:"

Reply:
Done